diff --unified --recursive --text package.orig/rcqt.so/src/action.cpp package.new/rcqt.so/src/action.cpp
--- package.orig/rcqt.so/src/action.cpp	2020-11-02 20:52:54.346675345 +0100
+++ package.new/rcqt.so/src/action.cpp	2020-11-02 20:53:23.430008727 +0100
@@ -196,13 +196,7 @@
   // Checking if version field exists
   string version_name;
   version_exists = has_property(VERSION, &version_name);
-  #if RVS_OS_TYPE_NUM == 1
-  string command_string = "dpkg --get-selections > ";
-  #endif
-  #if RVS_OS_TYPE_NUM == 2
-  string command_string = "rpm -qa --qf \"%{NAME}\n\" > ";
-  #endif
-  command_string += PKG_CMD_FILE;
+  string command_string = "pacman -Q > ";
 
   // We execute the dpkg-querry
   if (system(command_string.c_str()) == -1) {
