*** flang-rocm-4.5.0/CMakeLists.txt.bak	2021-11-19 18:39:54.679976196 +0100
--- flang-rocm-4.5.0/CMakeLists.txt	2021-11-19 18:40:12.911889722 +0100
***************
*** 375,384 ****
    add_definitions( -DFLANG_VERSION_MAJOR="${FLANG_VERSION_MAJOR} " )
    add_definitions( -DFLANG_VERSION_MINOR="${FLANG_VERSION_MINOR} " )
  endif()
! execute_process(COMMAND git log -1 --format=format:%H WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} OUTPUT_VARIABLE FLANG_SHA)
! string(STRIP ${FLANG_SHA} FLANG_SHA)
! message(STATUS "MY_VAR=${FLANG_SHA}")
! add_definitions( -DFLANG_SHA="Flang Sha ${FLANG_SHA} ")

  option(FLANG_INCLUDE_TESTS
         "Generate build targets for the Flang unit tests."
--- 375,381 ----
    add_definitions( -DFLANG_VERSION_MAJOR="${FLANG_VERSION_MAJOR} " )
    add_definitions( -DFLANG_VERSION_MINOR="${FLANG_VERSION_MINOR} " )
  endif()
! add_definitions( -DFLANG_SHA="Flang Sha None ")

  option(FLANG_INCLUDE_TESTS
         "Generate build targets for the Flang unit tests."
