diff --unified --recursive --text hipSPARSE-rocm-3.0/library/CMakeLists.txt hipSPARSE-rocm-3.0/library/CMakeLists.txt
--- hipSPARSE-rocm-3.0/library/CMakeLists.txt	2019-10-08 20:12:42.000000000 +0200
+++ hipSPARSE-rocm-3.0/library/CMakeLists.txt	2020-02-01 14:36:13.142332177 +0100
@@ -142,6 +142,8 @@
 
 set(HIPSPARSE_CONFIG_DIR "\${CPACK_PACKAGING_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}" CACHE PATH "Path placed into ldconfig file")
 
+include_directories("/opt/rocm/hsa/include")
+
 rocm_create_package(
   NAME ${package_name}
   DESCRIPTION "Radeon Open Compute SPARSE library"
