*** ROCm-OpenCL-Runtime-roc-3.5.0/amdocl/CMakeLists.txt.bak	2020-06-04 19:06:08.650037126 +0200
--- ROCm-OpenCL-Runtime-roc-3.5.0/amdocl/CMakeLists.txt	2020-06-04 19:04:34.824683968 +0200
***************
*** 109,111 ****
--- 109,114 ----
  find_package(Threads REQUIRED)
  
  target_link_libraries(amdocl64 amdrocclr_static Threads::Threads dl)
+ 
+ file(GENERATE OUTPUT "${CMAKE_BINARY_DIR}/amdocl64.icd" CONTENT "$<TARGET_FILE_NAME:amdocl64>")
+ install(FILES "${CMAKE_BINARY_DIR}/amdocl64.icd" DESTINATION "${CMAKE_INSTALL_SYSCONFDIR}/OpenCL/vendors/")
