diff --git a/CMakeLists.txt b/CMakeLists.txt
index 10430f3..25d0e1b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -89,7 +89,6 @@ endif()
 add_subdirectory(api/opencl/amdocl)
 add_subdirectory(compiler/lib/loaders/elf/utils/libelf)
 add_subdirectory(runtime)
-add_subdirectory(tools/clinfo)
 
 set(BUILD_SHARED_LIBS "Build shared libs" ON)
 add_subdirectory(api/opencl/khronos/icd)
@@ -101,9 +100,6 @@ include(ROCMSetupVersion)
 rocm_setup_version( VERSION "2.0.0" )
 
 # MAIN package
-install(PROGRAMS $<TARGET_FILE:clinfo>
-        DESTINATION bin/x86_64
-        COMPONENT MAIN)
 install(PROGRAMS $<TARGET_FILE:amdocl64>
         DESTINATION lib/x86_64
         COMPONENT MAIN)
